src/network/networkd-manager.c

```diff
@@ -488,6 +488,190 @@ int manager_rtnl_process_route(sd_netlink *rtnl, sd_netlink_message *message, vo
         return 1;
 }
 
+static int manager_rtnl_process_neighbor_lladdr(sd_netlink_message *message, union lladdr_union *lladdr, size_t *size, char **str) {
+        int r;
+
+        assert(message);
+        assert(lladdr);
+        assert(size);
+        assert(str);
+
+        *str = NULL;
+
+        r = sd_netlink_message_read(message, NDA_LLADDR, sizeof(lladdr->ip.in6), &lladdr->ip.in6);
+        if (r != -EIO) {
```

Comment:
Sorry, I meant this condition is not necessary.
Simply,
```
r = sd_netlink_message_read();
if (r >= 0) {
        ...
}
```
is enough.

Reply:
Oh, you want to continue trying other types even if we get a non-EIO error